### DISCUSSION TITLE
Welcome to AstraSolaris Discussions!

### DISCUSSION THREAD

**Discussion by SEscobedo** [SEscobedo/AstraSolaris#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by matheo** [comment]
# I'm Mateo Tibaquirá

I'm a Senior Angular Developer interested in a well-designed library to abstract all the layers and resources to animate the Solar System.

I want to be part of a community to learn and join efforts towards a nice tool useful to learn the concepts of our universe :)

**Reply by SEscobedo** [reply]
Welcome @matheo 
We will learn a lot of each other. 
I'm sure your contribution will be most valuable!
